File: Direct/iOS/SampleApp/SampleAppObjc/SampleAppObjc/MainViewController.m
//
//  MainViewController.m
//  SampleAppObjc
//
//  Created by Maria on 31/07/2023.
//

#import "MainViewController.h"
#import "MainViewHandler.h"
#import <CommonSampleApp/CommonSampleApp.h>

@implementation MainViewController

- (void)viewDidAppear:(BOOL)animated {
    [super viewDidAppear: animated];
    [self openMain];
}

- (void)openMain {
    [BaseMainViewController launchScreenWithViewController:self mainViewHandler:[[MainViewHandler alloc] init]];
}

@end

File: Direct/iOS/SampleApp/SampleAppSwift/SampleAppSwift/MainViewController.swift
//
//  MainViewController.swift
//  SampleApp
//
//  Created by Maria on 25/07/2023.
//

import Foundation
import UIKit
import CommonSampleApp

class MainViewController: UIViewController {
    
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        self.openMain()
    }
    
    func openMain() {
        BaseMainViewController.launchScreen(viewController: self, mainViewHandler: MainViewHandler())
    }
}
